.github/shared/test/spec-model.test.js

```diff
@@ -449,7 +449,7 @@ describe("getSwaggers", () => {
     const specModel = new SpecModel(folder, options);
     const swaggers = await specModel.getSwaggers();
     // Should return an array (may be empty if no valid readmes in this fixture)
-    expect(swaggers.length).toBe(5);
+    expect(swaggers.length).toBe(9);
```

Comment:
For clarity, please update test to check if `swaggers.map(s.path)` is equal to an array of expected strings, so we can see the paths.